### ISSUE TITLE
`request` 2.28.2 fails because of missing `charset_normalizer` files

### ISSUE THREAD

**Issue by kdeldycke** [Nuitka/Nuitka#2020]
Tl;Dr: `charset_normalizer` is missing an import that is only ever imported in a cythonized submodule. Thus, Nuitka cannot see it.

Binaries and properly generated by Nuitka, but their execution ends up with a traceback:

```pytb
Traceback (most recent call last):
  File "/tmp/onefile_1583_1674827414_883837/requests/compat.py", line 11, in <module requests.compat>
ModuleNotFoundError: No module named 'chardet'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/onefile_1583_1674827414_883837/__main__.py", line 60, in <module>
  File "/tmp/onefile_1583_1674827414_883837/__main__.py", line 54, in main
  File "/tmp/onefile_1583_1674827414_883837/meta_package_manager/__init__.py", line 33, in <module meta_package_manager>
  File "/tmp/onefile_1583_1674827414_883837/click_extra/__init__.py", line 66, in <module click_extra>
  File "/tmp/onefile_1583_1674827414_883837/click_extra/config.py", line 45, in <module click_extra.config>
  File "/tmp/onefile_1583_1674827414_883837/requests/__init__.py", line 45, in <module requests>
  File "/tmp/onefile_1583_1674827414_883837/requests/exceptions.py", line 9, in <module requests.exceptions>
  File "/tmp/onefile_1583_1674827414_883837/requests/compat.py", line 13, in <module requests.compat>
  File "/tmp/onefile_1583_1674827414_883837/charset_normalizer/__init__.py", line 24, in <module charset_normalizer>
  File "/tmp/onefile_1583_1674827414_883837/charset_normalizer/api.py", line 5, in <module charset_normalizer.api>
  File "/tmp/onefile_1583_1674827414_883837/charset_normalizer/cd.py", line 9, in <module charset_normalizer.cd>
ModuleNotFoundError: No module named 'charset_normalizer.md__mypyc'
Error: Process completed with exit code 1.
```

This is [also affecting PyInstaller](https://github.com/pyinstaller/pyinstaller-hooks-contrib/issues/534), but the [original author of `charset_normalizer` doesn't see a problem in this situation](https://github.com/Ousret/charset_normalizer/issues/253#issuecomment-1383234292). This issue is worth reporting here as it is a [direct dependency on the popular `request` package](https://github.com/psf/requests/issues/6331), and likely to be reported a lot by Nuitka users.

Should we add a workaround in Nuitka?

For the record, I reproduced this issue with the latest Nuitka v1.4:
```shell-session
$ nuitka --version
1.4
Commercial: None
Python: 3.10.9 (main, Dec 15 2022, 17:11:09) [Clang 14.0.0 (clang-1400.0.29.202)]
Flavor: Homebrew Python
Executable: /Users/kde/meta-package-manager/.venv/bin/python
OS: Darwin
Arch: arm64
Version C compiler: /usr/bin/clang (clang).
```

And for reference, here is the full [traceback as produced by my CI building my project](https://github.com/kdeldycke/meta-package-manager/actions/runs/4024929746/jobs/6917849266#step:4:23).

**Comment by kdeldycke** [comment]
Thanks @Maxwell175 for the validation that Nuitka is the right place to implement a workaround. Just proposed a PR at: https://github.com/Nuitka/Nuitka/issues/2020

**Comment by kayhayen** [comment]
This is on github develop right now, and will be in 1.4.1, I am putting the factory tag early because it will remind me to close the issue when the hotfix happens.

**Comment by kayhayen** [comment]
Part of the 1.4.2 hotfix I just made.
